kotlin/identify-layers/README.md

```diff
@@ -0,0 +1,34 @@
+# Identify layers
+
+Identify features in all layers in a map. 
+
+![Image of identify layers](identify-layers.png)
+
+## Use case
+
+IdentifyLayers allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
```

Comment:
minor readme improvement, we generally try to avoid calling API names in the use case: this one must have slipped through the common sample readme net!
```suggestion
Identify layers allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
```